gui/src/plotwidget.cpp

```diff
@@ -259,6 +264,33 @@ void PlotWidget::setUnitsVisible(bool visible)
 	}
 }
 
+void PlotWidget::printPlot()
+{
+	// select file where to save
+	QString selectedFilter;
+	QString fileName =
```

Comment:
Getting a file name should be separate from printing.
refactor this to printToFile(QString filename)

It should be possible to a QPixmap (or something similar) and have that be rendered in a file